oidc_example/op2/README.md

```diff
@@ -0,0 +1,40 @@
+# OP2 Example
+
+Getting Started
+---------------
+    git clone https://github.com/OpenIDC/pyoidc.git && cd pyoidc/oidc_example/op2/  
+    python3 -m venv venv && . venv/bin/activate  
+    pip install -r requirements.txt       
+    ./start.sh 
+### Setup
+#### Client Management
+
+To be able to start up the project you have to create a new client to the client_db.
+This is done by the CLI applilcation client_manageent.py
+The allowed redirect_uris must be given and the client_id, and client_secret are generated. 
```

Comment:
How on earth to do you get a diff like this, when these are the only changes you made?

![screenshot 2019-02-26 at 14 48 34](https://user-images.githubusercontent.com/786326/53417703-6ca44400-39d6-11e9-8582-fd3d9b13a759.png)